internal/querynode/flow_graph_delete_node.go

```diff
@@ -96,16 +97,20 @@ func (dNode *deleteNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 		)
 
 		if dNode.replica.getSegmentNum() != 0 {
-			processDeleteMessages(dNode.replica, delMsg, delData)
+			err := processDeleteMessages(dNode.replica, delMsg, delData)
```

Comment:
for delete, this might be a normal case?

Because each delete is forwarded to all querynode watch delta channel, but only on of the querynode should handle the delete.

Reply:
it has been filtered out in `processDeleteMessages`